jadx-core/src/main/java/jadx/core/codegen/MethodGen.java

```diff
@@ -180,8 +183,25 @@ private void addOverrideAnnotation(ICodeWriter code, MethodNode mth) {
 		}
 	}
 
+	private List<String> extractParamNamesFromAnnotation() {
+		AnnotationsAttr annotationList = mth.get(JadxAttrType.ANNOTATION_LIST);
+		if (annotationList != null) {
+			JadxAnnotation annotation = (JadxAnnotation) annotationList.get("Ldalvik/annotation/MethodParameters;");
```

Comment:
As soon as jadx now trying to decompile java bytecode and not only dex, dalvik specific annotations should be parsed in `jadx-dex-input` plugin and data class for common representation of such attribute should be placed in `jadx-plugins-api`.